server/repository/src/db_diesel/assets/asset_catalogue_item_row.rs

```diff
@@ -38,6 +39,7 @@ pub struct AssetCatalogueItemRow {
     pub model: String,
     #[column_name = "asset_catalogue_type_id"]
     pub type_id: String,
+    pub deleted_datetime: Option<chrono::NaiveDateTime>,
```

Comment:
Yes please, but didn't we say `is_active` for consistency?

Reply:
I know it's messy.. but.. I thought the discussion ended up saying that `deleted_datetime` is the preferred approach and we are using `is_active` to accomodate OG. in this case, we don't sync with OG, so we can use the preferred approach?

Reply:
I'll write up a wiki and get some review from the wider team.

Reply:
https://github.com/msupply-foundation/open-msupply/wiki/Code-Review-Comments---Rust#soft-delete